<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoToSleepSettingsBasic</class>
 <widget class="QDialog" name="GoToSleepSettingsBasic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="goToSleepTitle">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>16</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#BDBDBD</string>
   </property>
   <property name="text">
    <string>GoTo Sleep Time</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goToSleepTimeAccept">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>170</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color:#3B3B3B;
color:#BDBDBD;
border:none;
border-radius:3px;
}
QPushButton:pressed {
background-color:#606060;
color:#BDBDBD;
border:none;
border-radius:3px;
}
</string>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goToSleepTimeCancel">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>170</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color:#3B3B3B;
color:#BDBDBD;
border:none;
border-radius:3px;
}
QPushButton:pressed {
background-color:#606060;
color:#BDBDBD;
border:none;
border-radius:3px;
}
</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="smartStartTitle">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#616161</string>
   </property>
   <property name="text">
    <string>Smart Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goToSleepTimeIncrement">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>72</y>
     <width>66</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color:#3B3B3B;
color:#BDBDBD;
border:none;
border-radius:3px;
}
QPushButton:pressed {
background-color:#606060;
color:#BDBDBD;
border:none;
border-radius:3px;
}
</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goToSleepTimeDecrement">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>118</y>
     <width>66</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color:#3B3B3B;
color:#BDBDBD;
border:none;
border-radius:3px;
}
QPushButton:pressed {
background-color:#606060;
color:#BDBDBD;
border:none;
border-radius:3px;
}
</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="goToSleepTimeValue">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>55</y>
     <width>50</width>
     <height>44</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#2196F3</string>
   </property>
   <property name="text">
    <string>25</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="goToSleepTimeUnit">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>99</y>
     <width>50</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#616161</string>
   </property>
   <property name="text">
    <string>min</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QSlider" name="goToSleepTimeSlider">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>125</y>
     <width>217</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::handle:horizontal {
    background: #8f8f8f;
    width: 20px;
    border-radius: 7px;
}</string>
   </property>
   <property name="maximum">
    <number>45</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>349</width>
     <height>213</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 49, 49);
border: 1px;
border-radius: 5px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>goToSleepTitle</zorder>
  <zorder>goToSleepTimeAccept</zorder>
  <zorder>goToSleepTimeCancel</zorder>
  <zorder>smartStartTitle</zorder>
  <zorder>goToSleepTimeIncrement</zorder>
  <zorder>goToSleepTimeDecrement</zorder>
  <zorder>goToSleepTimeValue</zorder>
  <zorder>goToSleepTimeUnit</zorder>
  <zorder>goToSleepTimeSlider</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
